config/initializers/rollbar.rb

```diff
@@ -0,0 +1,8 @@
+require 'rollbar/rails'
+Rollbar.configure do |config|
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.